lib/_debugger.js

```diff
@@ -65,15 +65,13 @@ function Protocol() {
 }
 exports.Protocol = Protocol;
 
-
 Protocol.prototype._newRes = function(raw) {
-  this.res = { raw: raw || '', headers: {} };
+  this.res = {raw: raw || '', headers: {}};
```

Comment:
I feel like it's more common style to leave blank space on `{ }`. Could we leave that?